.bazelci/presubmit.yml

```diff
@@ -24,3 +24,100 @@ platforms:
     - "--enable_runfiles"
     test_targets:
     - "//..."
+tasks:
+  web_testing_go_ubuntu2004:
+    name: "Go workspace for web in ubuntu"
```

Comment:
I think "workspace" is no longer used.
Maybe "rules_web_testing_go in ubuntu" (and others)